src/modules/core/components/Comment/Actions/CommentActions.tsx

```diff
@@ -1,17 +1,14 @@
 import React, { useState, useEffect } from 'react';
 import { defineMessages } from 'react-intl';
 
-import classnames from 'classnames';
 import { COMMENT_MODERATION } from '~immutable/index';
-import Icon from '~core/Icon';
 import Popover from '~core/Popover';
+import ThreeDotsButton from '~core/Button/ThreeDotsButton';
```

Comment:
We should add a `ThreeDotsButton` export in the `index` file so that we can import the same way as the other buttons:
`import { ThreeDotsButton } from '~core/Button';`

Reply:
@ArmandoGraterol consider it done. Thanks!